db/backup.go

```diff
@@ -90,7 +91,8 @@ func (db *db) basicImport(ctx context.Context, txn datastore.Txn, filepath strin
 				return NewErrDocFromMap(err)
 			}
 
-			err = col.WithTxn(txn).Create(ctx, doc)
+			// TODO=ACP: https://github.com/sourcenetwork/defradb/issues/2430 - Add identity ability to backup
```

Comment:
As we discussed in one of the standups that we want the admin to be able to backup everything including private docs, and that ability to turn off acp is only programmatic until admin identity is implemented.

So if an owner does disable acp programmatically then backup will backup everything and restore will make everything public. 

If acp is enabled (the default right now) and there were private documents then backup will only backup public documents.

- [ ] Add warning / more info on using backup <> acp until https://github.com/sourcenetwork/defradb/issues/2430

Reply:
> Note that this backup functionnality should eventually be disabled for collections/documents that are protected by ACP since any update to documents will cause their docIDs to be different at import. This would most likely break ACP.

Are you saying we want to disable backup permanently for collections/docs that are being access controlled? OR just that the current implementation of backup would need to be removed until one that supports ACP is implemented?

> Alternatively we can allow usage with the caveat that importing will have unexpected behaviour.

The only caveat right now is that it will only backup public documents.

Reply:
Makes sense!

I will add a caveat section in ACP documentation and link it to the user facing backup documentation for now

- [ ] Caveat / Warning section

Reply:
Added warning in commit: ``` PR(ACP-DOCS): Warning / Caveat Section In README ```